### PR TITLE
Decompile module 52

### PR DESCRIPTION
**Pull request by remicalixte**
This one looks like it's called when creating a new save file. SetAvatar and SetTrainer_ID makes me think so.

No idea what is the big byte array at the end

### CODE REVIEW

File: include/global.h [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 #define GF_ASSERT(expr) do {if (!(expr)) ErrorHandling();} while (0)
 
+#define UNUSED __attribute__((unused))
+
```
Comment (red031000) [inline include/global.h:18]: this isn't valid, mwcc does not use \_\_attribute\_\_. if you want to use unused, do #pragma unused(var)

**Review by red031000 — CHANGES_REQUESTED** [review]
module 52 really should be split into its constituent files, blame me for not being fast enough on that

**Comment by remicalixte** [comment]
How can I know the correct split?

**Comment by red031000** [comment]
the correct split is defined by the japanese version, the compiler used there ordered functions differently in files (last first) compared to the US version

**Comment by remicalixte** [comment]
So I need to extract a japanese rom to find out the split? Is there a tool to do this?

**Comment by red031000** [comment]
don't worry about it for now, I can sort it out later

**Review by red031000 — APPROVED** [review]

**Pull request merged**